README.md

```diff
@@ -50,4 +50,15 @@ Nevertheless, the order of the user stories in each group does not reflect the d
 - As a system administrator, I would like the players to have an enjoyable experience when interacting with the system.
 
 ## Engineering Requirements
-While it might be slightly advantageous if your solution involves technologies/frameworks/programming languages in our tech stack,you are allowed to use any technology/framework/programming language you prefer or are most familiar with. Regardless of which you eventually use, we expect the code to be in high quality.
+While it might be slightly advantageous if your solution involves technologies/frameworks/programming languages in our tech stack:
+- python
+- javascript (es6)
+- typescript
+- golang
+- java
+- groovy
+- docker
+
+you are welcome to use any technology/framework/programming language you prefer or are most familiar with. Regardless of which you eventually use, we expect the code to be in *high quality*.
+
+We also love marrying up engineering excellence with great UX/UI design, demostrating your tenant in frontend work will definitely be a bounus!
```

Comment:
While it's entirely up to @soloman1124, I personally don't think UI/UX should be here. We have Matthew for UI/UX already, so I think we wouldn't mind if the candidate produces a bad UI. As a result, having this here, we are only stressing people out with things that we don't really care.

Reply:
The frontend work could potentially be as a user stories in Could Have: As a player, I would like to be able to play the game anywhere with any device through an Internet browser.

Reply:
Well, I think you prefer more formal spec type of writing. I find that thing about the device browser etc is a bit lengthy and not very intuitive to me...

Reply:
Adding on to the UI/UX thing, if @soloman1124 sees it as a secondary skill, I think it usually emerges when a person is creating a UI, like Andrew Hill and Vivian. So it isn't necessary unless we are specifically looking for people with strong UI/UX skills and expect them to do some design work when they get hired.

Reply:
I also don’t feel it is an requirement, it’s just a hint if they want to do more work to impress us

Reply:
For the browser thingy, I think it's more like the curse of knowledge... Terms like "tenant" and "front-end work" may not necessarily make sense to every graduate developers. If we expect the candidate to have web development experience, perhaps we can paraphrase it to "As a player, I would like to be able to play the game on a website."